src/main/java/ru/yandex/practicum/filmorate/model/User.java

```diff
@@ -0,0 +1,22 @@
+package ru.yandex.practicum.filmorate.model;
+
+import javax.validation.constraints.*;
+import lombok.Data;
+import ru.yandex.practicum.filmorate.annotations.CorrectLogin;
+
+import java.time.LocalDate;
+
+@Data
+public class User {
+    @PositiveOrZero(message = "id can not be negative")
+    private int id; // целочисленный идентификатор
+    @NotNull(message = "login must not be null")
+    @Email(message = "invalid email")
+    private String email; // электронная почта
+    @NotBlank(message = "login must not be empty") // @NotBlank - элемент не должен быть нулевым и должен содержать хотя бы один непробельный символ
+    @CorrectLogin
+    private String login; // логин пользователя
+    private String name; // имя для отображения
+    @PastOrPresent // Аннотированный элемент должен быть моментом, датой или временем в прошлом или настоящем
```

Comment:
```suggestion
      /**
     * Дата рождения должна быть моментом, датой или временем в прошлом или настоящем
     */
     @PastOrPresent
```

А вот так java-doc выглядит в коде:
https://disk.yandex.ru/i/sAr-xGKAorQoig 
https://disk.yandex.ru/i/6aktM_P8LchN-A